core-tools.markdown

```diff
@@ -0,0 +1,345 @@
+---
+title: Learn about the core tools
+page: core-tools
+isDocumentation: true
+---
+
+# Learn about the core tools
```

Comment:
Given the audience for this article, I wonder if we wouldn't be better to actually pick an installation method and walk through it to start with (or we can have that as another article and link back to it). I know that picking a single installation method will be controversial, but it seems like this document is suffering a lot from the lack of being able to assume a specific configuration.

Reply:
The main thing is that I think the prose sometimes becomes a bit hard to follow because of the need to talk around multiple setups. Being able to say: "Type `xyz` into your terminal" is a powerful thing in terms of making the tutorial easy for people to follow. As a secondary issue, I think if we're aiming this at early beginners, the fact that there are many choices is going to be overwhelming to them.

Reply:
I think I could understand better if you could please mention places where you think the prose becomes a bit hard to follow and could be improved.

I don't quite understand because except for one instance (using packages) this tutorial does not talk about multiple setups. And it does not asks the users to make choices.

Reply:
@rebeccaskinner I believe installation instructions have no place here, that's what https://www.haskell.org/downloads/ is for